src/documents/HowToUseTheRenderer.md

```diff
@@ -60,10 +60,10 @@ For detailed information on how to use this function, see [fluid.selfRender](htt
 
 In addition to these primary ways of using the Renderer, there are a several other functions that are useful in certain circumstances. These are described here.
 
-### `fluid.rerender` ###
+### fluid.rerender ###
```

Comment:
The name of this function is actually "fluid.reRender" with a capital R in the middle